src/command_modules/azure-cli-resource/tests/test_locks.py

```diff
@@ -0,0 +1,39 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+
+
+class StorageAccountTests(ScenarioTest):
+    def test_list_locks(self):
+        locks_list = self.cmd('az lock list').get_output_in_json()
```

Comment:
I would recommend leaving the test to verify that you don't get a CLI error or SDK error but just not checking anything. This verifies that the minimum call (required params only) works.